### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by MadsHT**
added the ability to disable services and changed some of the urls

### CODE REVIEW

**Comment by MadsHT** [comment]
Made some changes:

- the ability to enable services
- changed the yggdrasil repo url to point at olympus
- disabled CNI by default

File: yggdrasil/templates/ingress.yaml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {{- $root := . -}}
 {{- range $path, $_ := .Files.Glob "**/**/config.yaml" }}
   {{ $project := $root.Files.Get $path | fromYaml }}
+  {{ if (index $root.Values $project.name).enabled }}
+
```
Comment (jaxels10) [inline yggdrasil/templates/ingress.yaml:5]: Slet ekstra linje

File: yggdrasil/templates/namespace.yaml [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
 {{- range $path, $_ := .Files.Glob "**/**/config.yaml" }}
-    {{ $project := $root.Files.Get $path | fromYaml }}
+  {{ $project := $root.Files.Get $path | fromYaml }}
+  {{ if (index $root.Values $project.name).enabled }}
 ---
 apiVersion: v1
 kind: Namespace
 metadata:
   name: {{ $project.namespace }}
+
```
Comment (jaxels10) [inline yggdrasil/templates/namespace.yaml:11]: Slet ekstra linje

File: yggdrasil/templates/project.yaml [hunk 3]
```diff
@@ -2,6 +2,8 @@
 {{- range $path, $_ := .Files.Glob "**/**/config.yaml" }}
     {{- with $root }}
         {{ $project := .Files.Get $path | fromYaml }}
+        {{ if (index $root.Values $project.name).enabled }}
+{{- if $project.projectName }}
```
Comment (jaxels10) [inline yggdrasil/templates/project.yaml:6]: Har faktisk slettet denne if igen fra min branch, så slet den lige og fjern også {{ end }} fra den.

**Review by jaxels10 — CHANGES_REQUESTED** [review]

**Review by jaxels10 — APPROVED** [review]

**Pull request merged**